tmk_core/common.mk

```diff
@@ -164,6 +164,7 @@ ifeq ($(strip $(BLUETOOTH_ENABLE)), yes)
 endif
 
 ifeq ($(strip $(BLUETOOTH)), AdafruitBLE)
+	SRC += analog.c
```

Comment:
`converter/usb_usb/ble` is still broken according to Travis. Instead of adding analog.c here, it should actually be added to `LUFA_SRC` in the relevant block, and `drivers/avr` included in the search path:

```diff
diff --git a/tmk_core/common.mk b/tmk_core/common.mk
index 9bc4742f4..221688755 100644
--- a/tmk_core/common.mk
+++ b/tmk_core/common.mk
@@ -164,7 +164,6 @@ ifeq ($(strip $(BLUETOOTH_ENABLE)), yes)
 endif
 
 ifeq ($(strip $(BLUETOOTH)), AdafruitBLE)
-	SRC += analog.c
 	TMK_COMMON_DEFS += -DBLUETOOTH_ENABLE
 	TMK_COMMON_DEFS += -DMODULE_ADAFRUIT_BLE
 	TMK_COMMON_DEFS += -DNO_USB_STARTUP_CHECK
diff --git a/tmk_core/protocol/lufa.mk b/tmk_core/protocol/lufa.mk
index bb82a31e1..d7d1d9ffc 100644
--- a/tmk_core/protocol/lufa.mk
+++ b/tmk_core/protocol/lufa.mk
@@ -29,6 +29,7 @@ ifeq ($(strip $(BLUETOOTH_ENABLE)), yes)
 endif
 
 ifeq ($(strip $(BLUETOOTH)), AdafruitBLE)
+		LUFA_SRC += analog.c
 		LUFA_SRC += $(LUFA_DIR)/adafruit_ble.cpp
 endif
 
@@ -51,6 +52,7 @@ SRC += $(LUFA_SRC)
 # Search Path
 VPATH += $(TMK_PATH)/$(LUFA_DIR)
 VPATH += $(LUFA_PATH)
+VPATH += $(DRIVER_PATH)/avr
 
 # Option modules
 #ifdef $(or MOUSEKEY_ENABLE, PS2_MOUSE_ENABLE)
```

Now both of them compile, at least. Sorry about that.

Reply:
That works for me. Thanks again.